content/en/apps/examples/pharmacovigilance-reference-app.md

```diff
@@ -0,0 +1,63 @@
+
+## Solution Overview
+
+The CHT Pharmacovigilance Reference app is designed and developed by [IntelliSOFT Consulting LTD](https://www.intellisoftkenya.com/). Components of Kenya’s Pharmacovigilance Electronic Reporting System have been integrated into the Community Health Toolkit (CHT) to support community health providers (CHPs) and Community Health Assistants (CHAs) to efficiently identify, report, refer and manage adverse drug reactions and AEFI while delivering healthcare. 
+
+In the spirit of openness and as a contribution to the CHT open source project, Medic and IntelliSOFT have coordinated to release the source code and the workflows documentation of the CHT Pharmacovigilance Reference app.
+This reference app provides an example app that other CHT Implementers can learn how they can build pharmacovigilance workflows to help improve patient safety for community members they serve
+
+## Key Pharmacovigilance Areas Supported
+
+ 1. Adverse Drug Reactions (ADRs): detect and report ADRs resulting from pharmaceutical products.
+ 2. Adverse Effects Following Immunization (AEFI): monitor and report adverse effects following immunization.
+ 3. Poor Quality Health Products and Technologies: identify and report incidents related to the use of poor-quality health products and technologies
+ 4. Death  Monitoring: Monitor, identify, report and record deaths resulting from monitored conditions within household members in the Pharmacovigilance application
+  
+
+## Users and hierarchy
+
+```
+insert diagram
+```
+
+## Pharmacovigilance workflows
+
+### Pharmacovigilance workflow that supports reporting, referral and follow up of patients
+
+During a household visit, a CHP uses the pharmacovigilance app to assess household members for suspected severe adverse reactions from medication usage or recent vaccinations/immunisations. Using the assessment form a CHP records any suspicious issues   with the medication, such as unusual colour, smell, or packaging and adverse drug effects from medication and immunizations.
+
+A public report task is generated for a CHA if a CHP submits a suspected case of adverse reaction. The public report form captures information about the adverse effects and medications including side effects, onset date, and current status of the reaction. Additionally, the form captures medication details such as name of the medicine, manufacturer, purchase location, start date, stop date, and expiry date. In case of multiple medications, the form allows a CHA  to enter more details about each medication. 
```

Comment:
```suggestion
A public report task is generated for a CHA once a CHP submits a suspected case of adverse reaction. The public report form captures information about the adverse effects and medication including side effects, onset date, and current status of the reaction. Additionally, the form captures medication details such as name of the medicine, manufacturer, purchase location, start date, stop date, and expiry date. In case of a variety of medicines, the form allows a CHA to perform multiple entries. 
```